src/explore-education-statistics-admin/src/services/releasePermissionService.ts

```diff
@@ -0,0 +1,40 @@
+import client from '@admin/services/utils/service';
+
+export interface ManageAccessPageContributor {
+  userId: string;
+  userFullName: string;
+  userEmail: string;
+  releaseRoleId: string | undefined;
+}
+
+const releasePermissionService = {
+  getPublicationReleaseContributors(
+    publicationId: string,
+    releaseId: string,
+  ): Promise<ManageAccessPageContributor[]> {
+    return client.get(`/publications/${publicationId}/contributors`, {
+      params: { releaseId },
+    });
+  },
+  getAllPublicationContributors(
+    releaseId: string,
```

Comment:
Similar thing to the backend - this service method doesn't make much sense as it should be receiving some `publicationId`, not a `releaseId`.